dolphinscheduler-task-plugin/dolphinscheduler-task-dvc/src/main/java/org/apache/dolphinscheduler/plugin/task/dvc/DvcParameters.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.task.dvc;
+
+import org.apache.dolphinscheduler.plugin.task.api.parameters.AbstractParameters;
+
+import java.util.HashMap;
+
+public class DvcParameters extends AbstractParameters {
+
+    /**
+     * common parameters
+     */
+
+    private String dvcTaskType;
```

Comment:
Thanks for your suggestion, I'll think about how to fix it.

Reply:
> It is recommended to use an enumeration class.

I looked at the code again, and if I changed it to an enumerated type, it seemed less readable between the front-end and back-end.
 I have defined constants of this type in the `DvcConstants.java` file. I don't know if it's okay to do that.
<img width="553" alt="image" src="https://user-images.githubusercontent.com/31528124/174555037-79b091ea-8ada-4f02-b45b-1b53c405b5c4.png">
What do you think? @caishunfeng @zhuangchong

Reply:
If it is changed to enum, is it difficult to maintain if need to add more enum?

Reply:
There seems to be less flexibility between the front-end and back-end transfer than using string directly.

Reply:
<img width="968" alt="image" src="https://user-images.githubusercontent.com/31528124/174560364-03205a31-565d-43ac-9fed-3d224eda8b79.png">

If it changed to enum, it seems that the value of this field passed in by the front end needs to be additionally converted to type enum

Reply:
It's okay to use strings as you are doing now.

Reply:
In fact, my suggestion is to use enumeration, you don't need to make any changes in the front end you write, use the enumeration class in the backend `TaskParamter` to receive `dvgTaskType`
``` 
private DvcTaskTypeEnum dvcTaskType 
```
Whether it is developed according to the above, its impact is minimal.

Reply:
I have changed the dvcTaskType to enumeration. Please help me to see if it is ok for me to write this way because I am not very familiar with Java, but after I test the modification, there is no problem with the program. @zhuangchong